### DISCUSSION TITLE
How to document rate limit in Swagger?

### DISCUSSION THREAD

**Discussion by sarahhachouche1** [ThreeMammals/Ocelot#2027] [Q&A] [answered]
```json
"RateLimitOptions": {
  "EnableRateLimiting": true,
  "Period": "1s",
  "PeriodTimespan": 1,
  "Limit": 10,
  "ClientWhitelist": [ "9e32fca1-df87-4463-8ef0-cf5670186687" ]
}
```
How can I document the response for quota in Swagger?

**Comment by raman-m** [comment, marked as answer]
Dear Sarah,

Ocelot doesn't officially support Swagger! See the link: [Not Supported | Swagger](https://ocelot.readthedocs.io/en/latest/introduction/notsupported.html#swagger)
Basic support of Swagger will be added in the 2nd part of 2024...

How did you integrate Swagger into your Ocelot app?

**Reply by sarahhachouche1** [reply]
i used ReConfigureUpstreamSwaggerJson option to write this, i used this documentation https://medium.com/@sanchit.bhardwaj31/using-swagger-with-ocelot-api-gateway-11a5fa89ff12

**Reply by raman-m** [reply]
@sarahhachouche1 
Quite old, but simple instructions for beginners. Thanks for sharing!

How useful is [MMLib.SwaggerForOcelot](https://github.com/Burgyn/MMLib.SwaggerForOcelot) package for you?
What do you like and what don't you like in this library?

@Burgyn FYI
